.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+#
+name: Create and publish the Ground Station Docker image
+
+# Configures this workflow to run every time a change is pushed to the branch called `release`.
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+
+# Defines two custom environment variables for the workflow. These are used for the Container registry domain, and a name for the Docker image that this workflow builds.
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
```

Comment:
Can you change the image name to be ground-server, not ground-software? Will be important if we have more docker images in the future.